WebAPI/Controllers/OrganizationsController.cs

```diff
@@ -0,0 +1,139 @@
+using EntityModel;
+using Microsoft.AspNetCore.Mvc;
+using Shared;
+using Shared.ApiInterface;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using WebAPI.Models;
+
+namespace WebAPI.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class OrganizationsController : ControllerBase
+    {
+        protected readonly IApiInterface api;
+
+        public OrganizationsController(EfInterface api) => this.api = api ?? throw new ArgumentNullException(nameof(api));
+
+        // GET: api/organizations?name={}&lat={}&lon={}&services={}&distance={}
+        [HttpGet]
+        public async Task<ActionResult<List<OrganizationDTO>>> Get(string name = "", string lat = "", string lon = "", string services = "", double maxDistance = 25)
+        {
+            var organizations = await api.GetVerifiedOrganizations();
+
+
+            if (organizations != null)
+            {
+                var orgsDTO = new List<OrganizationDTO>();
+
+                if (!String.IsNullOrEmpty(name))
+                {
+                    organizations = organizations.Where(o => o.Name == name).ToList();
+                }
+                if (!(String.IsNullOrEmpty(lat) && String.IsNullOrEmpty(lon)))
```

Comment:
I think you want if both are not empty here - if one is empty and this other is not, this will still pass